google-cloud-bigquery/lib/google/cloud/bigquery/service.rb

```diff
@@ -473,6 +473,7 @@ def query_config query, options = {}
         def resolve_legacy_sql legacy_sql, standard_sql
```

Comment:
I agree it borders on too much duplication, but this is a private method and can be refactored in the future without affecting the public API. I'd rather feel the pain of the duplication before DRYing it, and right now its not causing me any pain.